src/website/src/app/documentation/documentation.component.html

```diff
@@ -3,7 +3,7 @@
     <section class="sidenav-content">
         <version-switcher></version-switcher>
 
-        <a class="nav-link" routerLink="documentation" routerLinkActive="active"
+        <a class="nav-link" routerLink="/documentation" routerLinkActive="active"
            [routerLinkActiveOptions]="{exact:true}">Component Status</a>
 
         <a class="nav-link" routerLink="get-started" routerLinkActive="active"
```

Comment:
Any reason why `/documentation` gets a root slash, but `get-started` and the other links don't?

Reply:
I believe since /documentation is a lazy loaded, but also root route and the rest are relative to it.